app/Services/ExportService.php

```diff
@@ -55,13 +55,13 @@ public function exportSignups($campaignId, $campaignRunId)
                 'campaign_id' => $signup->campaign_id,
                 'campaign_run_id' => $signup->campaign_run_id,
                 'northstar_id' => $signup->northstar_id,
-                'first_name' => $northstarUser->first_name,
-                'last_name' => $northstarUser->last_name,
-                'email' => $northstarUser->email,
-                'mobile' => $northstarUser->mobile,
-                'zip_code' => $northstarUser->addr_zip,
-                'birthdate' => $northstarUser->birthdate,
-                'age' => getAgeFromBirthdate($northstarUser->birthdate),
+                'first_name' => $northstarUser->first_name ?? 'N/A',
```

Comment:
@sbsmith86 I've never seen `??` before - so cool! Curious how it works? (I tried to google php ?? and all I got were articles about what php is haha)

Reply:
I had the same thought this shortcut is awesome!!!

Reply:
I found something that uses `?:` in a similar way, do you know if the two differ at all?

Reply:
ah yes yes https://en.wikipedia.org/wiki/Null_coalescing_operator#PHP

Reply:
PHP 7 specific tho..

Reply:
@katiecrane the main difference is that `??` checks `isset()` instead of just the truthy-ness of a thing.

Reply:
so cool!